ASTProcessor/src/test/java/edu/usb/argos/ASTProcessor/FileReaderTest/AntlrFileReaderByTextTest.java

```diff
@@ -0,0 +1,58 @@
+package test.java.edu.usb.argos.ASTProcessor.FileReaderTest;
+
+import edu.usb.argos.ASTProcessor.reader.application.services.FileReaderByText;
+
+import org.antlr.v4.runtime.tree.ParseTree;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
It's not good to use * when importing due to we access to the all library so please import just what you need.